.github/workflows/windows-ci.yml

```diff
@@ -6,12 +6,12 @@ on:
       os-version:
         description: 'OS version to run the workflow'
         required: false
-        default: 'windows-13'
+        default: 'windows-2019'
         type: string
       vs-toolset:
         description: 'Visual Studio toolset to use'
         required: false
-        default: 'v142'
+        default: ''
```

Comment:
```suggestion
        default: 'v142'
```

Reply:
I've removed the default vs-toolset because setting a toolset will make the workflow run two different jobs with the same toolset as this one is a matrix running for v142 and v141. By leaving it empty by default, we run both jobs with default toolsets. 

Added better description in 0c21dd4